courses-rest/src/main/java/no/sikt/nva/fs/CoursesByInstitutionOfLoggedInUserHandler.java

```diff
@@ -1,43 +1,90 @@
 package no.sikt.nva.fs;
 
+import static nva.commons.core.attempt.Try.attempt;
 import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import java.net.HttpURLConnection;
+import java.net.URI;
+import java.time.Clock;
+import java.time.ZoneId;
+import java.util.Optional;
+import no.sikt.nva.fs.config.FsConfig;
+import no.sikt.nva.fs.config.InstitutionConfig;
+import no.unit.nva.commons.json.JsonUtils;
 import nva.commons.apigateway.ApiGatewayHandler;
 import nva.commons.apigateway.RequestInfo;
 import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class CoursesByInstitutionOfLoggedInUserHandler extends ApiGatewayHandler<Void, CoursesByInstitutionResponse> {
-
-    private static final Logger logger = LoggerFactory.getLogger(CoursesByInstitutionOfLoggedInUserHandler.class);
+import nva.commons.core.attempt.Try;
+import nva.commons.core.paths.UriWrapper;
 
+public class CoursesByInstitutionOfLoggedInUserHandler extends ApiGatewayHandler<Void, Course[]> {
```

Comment:
it is better to return a typed object that contains the array as a field.

Reply:
Adding an extra layer of abstraction? Why is it better?

Reply:
One reason to return an object, is that when we are returning an object, we can more fields, should we found this necessary, e.g. number of courses, or information about the institution or if it is too many courses pagination info. 

The other reason is that NVA's  target is to use Linked Data so that we can describe the information we expose through ontologies. The data will be linked to a context (i.e. an ontology that will explain the meaning of each field) This is already done in Publication API and in other services and the target is to be done everywhere in NVA. In order to do that we need to return a typed object, because an array is not an object.